packages/app/obojobo-express/server/migrations/20240130162639-modify-users-add-last-login.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+
+var dbm
+var type
+var seed
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function(options, seedLink) {
+	dbm = options.dbmigrate
+	type = dbm.dataType
+	seed = seedLink
+}
+
+exports.up = function(db) {
+	return db.addColumn('users', 'last_login', {
+		type: 'timestamp WITH TIME ZONE',
+		notNull: true,
+		defaultValue: new String('now()')
+	})
+}
+
+exports.down = function(db) {
+	return db.removeColumn('last_login')
```

Comment:
`db.removeColumn` takes two arguments, the first being the table name.

This should read `return db.removeColumn('users', 'last_login')`.